{{cookiecutter.repo_name}}/config/settings.py

```diff
@@ -94,10 +97,16 @@
     {%- endif %}
 )
 
+if env('DJANGO_USE_SILK'):
```

Comment:
@stepanovsh I think it is better to move it right after `LOCAL_APPS`. Cause in current way main project's apps are devided by debug app with `IF state`